Source/Atomic/Resource/Image.cpp

```diff
@@ -2034,7 +2124,12 @@ unsigned char* Image::GetImageData(Deserializer& source, int& width, int& height
 
     SharedArrayPtr<unsigned char> buffer(new unsigned char[dataSize]);
     source.Read(buffer.Get(), dataSize);
-    return stbi_load_from_memory(buffer.Get(), dataSize, &width, &height, (int*)&components, 0);
+    // #623 BEGIN TODO: Crunch expects all 4 components. This isn't ideal, but since
+    // precompressed versions should always be loaded anyway it doesn't matter.
+    // Might need to handle differently per platform.
+    components = 4;
+    return stbi_load_from_memory(buffer.Get(), dataSize, &width, &height, nullptr, components);
+    // #623 END TODO: Crunch expects all 4 components. This isn't ideal, but since

```

Comment:
The only other call point is LoadColorLUT, but I can introduce a GetImageDataForceComponents and use that in BeginLoad guarded by a windows check? BTW BeginLoad is a serious refactoring candidate :) I didn't want to do it because I wasn't sure if it was originally an Urho class, and I know merging something like that can be a royal PITA.